src/main/java/com/google/api/codegen/GeneratorVersionProvider.java

```diff
@@ -24,14 +23,15 @@
   public static String getGeneratorVersion() {
     String version = DEFAULT_VERSION;
     Properties properties = new Properties();
-    try {
-      properties.load(
-          GeneratorVersionProvider.class
-              .getResourceAsStream("/com/google/api/codegen/codegen.properties"));
-      version = properties.getProperty("version");
-    } catch (IOException e) {
-      e.printStackTrace(System.err);
-    }
-    return version;
+    return "0.0.5";
+    //    try {
+    //      properties.load(
+    //          GeneratorVersionProvider.class
+    //              .getResourceAsStream("/com/google/api/codegen/codegen.properties"));
+    //      version = properties.getProperty("version");
+    //    } catch (IOException e) {
+    //      e.printStackTrace(System.err);
+    //    }
+    //    return version;
```

Comment:
We should just delete this code. The right way to get the version is through a metadata file.

Reply:
Reverting this back to master. It was just a hack for when intellij couldn't find the properties file.